edison/src/main/java/edu/illinois/cs/cogcomp/edison/features/factory/ClauseFeatureExtractor.java

```diff
@@ -45,7 +45,11 @@ public ClauseFeatureExtractor(String pseudoParseViewName, String clauseViewName)
     @Override
     public Set<Feature> getFeatures(Constituent c) throws EdisonException {
         TextAnnotation ta = c.getTextAnnotation();
-        TreeView pseudoParseView = (TreeView) ta.getView(parseViewName);
+        TreeView pseudoParseView;
+        // If the real parse view exists, use that
```

Comment:
could you add a little clarification here why the pseudo-parse might not exist, while the real parse exists?